src/Microsoft.DotNet.Build.Tasks.Feed/SleetLogger.cs

```diff
@@ -0,0 +1,70 @@
+﻿using NuGet.Common;
+using System.Threading.Tasks;
+using MSBuild = Microsoft.Build.Utilities;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    public class SleetLogger : ILogger
+    {
+        private MSBuild.TaskLoggingHelper _log;
+
+        public SleetLogger(MSBuild.TaskLoggingHelper log)
+        {
+            _log = log;
+        }
+        public void Log(LogLevel level, string data)
+        {
+            _log.LogMessage(data, level);
+        }
+
+        public void Log(ILogMessage message)
+        {
+            _log.LogMessage(message.Message, message.Level);
+        }
+
+        public Task LogAsync(LogLevel level, string data)
+        {
+            return Task.Run(() => _log.LogMessage(data, level));
```

Comment:
[In NugetMsBuildLogger](https://github.com/dotnet/buildtools/blob/9bc085cef38e12857f93a94ba2c9fd1faef4052a/src/Microsoft.DotNet.Build.Tasks/NugetMsBuildLogger.cs#L96) we do the logging then return `Task.CompletedTask`. Is there a reason to use `Task.Run` here?

Reply:
No reason in particular. When I tried to change it to Task.CompletedTask it failed at run time claiming the property did not exist hence left it as Task.Run